src/review-factoring.ptx

```diff
@@ -613,50 +613,55 @@
 </subsection>
 
 <subsection>
-    <!-- I can't get the angles and sizes to work perfectly... Alex will you fix it so the perfect squares don't align. -->
     <title>Flow Chart for Factoring</title>
         <p>Deciding which method to use when factoring a random polynomial can seem like a daunting task. Understanding all of the techniques that we have learned and how they fit together is best done using a flow chart.</p>
                 <figure xml:id="figure-factoring-flow-chart">
                 <caption>Factoring Flow Chart</caption>
                     <image>
                         <description>A decision tree to help factoring.</description>
                         <latex-image-code>
-                            <!-- todo: ross is still working on this stupid thing. -->
+                            <!-- todo: ross is still working on this stupid thing. Now it won't compile at all...-->
```

Comment:
Ross, let's not spend more time on this. I'll take what you've done so far and work on it at some point. Certainly I will see anything not working or coming out funky when I process the PDF.